rust/kcl-lib/src/std/shapes.rs

```diff
@@ -174,7 +153,7 @@ pub async fn circle_three_point(exec_state: &mut ExecState, args: Args) -> Resul
 #[stdlib {
     name = "circleThreePoint",
     keywords = true,
-    unlabeled_first = true,
+    unlabeled_first = false,
```

Comment:
This function uses the unlabeled first, so is this a mistake?

Reply:
Ah, I think I was confused by the arg order and that had some knock-on effect somewhere. I'll revert this and see if anything needs fixing up